src/app/components/Feedback/Feedback.jsx

```diff
@@ -0,0 +1,214 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import FocusTrap from 'focus-trap-react';
+
+import appConfig from '../../../../appConfig';
+
+
+class Feedback extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      showForm: false,
+      feedback: '',
+      email: '',
+      success: null,
+    };
+
+    this.feedbackField = React.createRef();
+
+    this.onSubmitForm = this.onSubmitForm.bind(this);
+    this.openForm = this.openForm.bind(this);
+    this.closeForm = this.closeForm.bind(this);
+    this.deactivateForm = this.deactivateForm.bind(this);
+    this.handleFeedbackChange = this.handleFeedbackChange.bind(this);
+    this.handleEmailChange = this.handleEmailChange.bind(this);
+    this.sendFeedback = this.sendFeedback.bind(this);
+    this.handleRadioChange = this.handleRadioChange.bind(this);
+  }
+
+  onSubmitForm(e) {
```

Comment:
Probably needs some sort of error message if you submit the form without the required field
(I'd argue that we probably don't need to require comments, though.  Like, it's useful even to say "Yes I found what I needed")

Reply:
Added in the new commits I just pushed!